qgreenland/util/luigi/tasks/fetch.py

```diff
@@ -16,14 +16,14 @@
 
 # TODO: call this 'FetchDataset'? 'FetchAsset'?
 class FetchTask(luigi.Task):
-    dataset_cfg = luigi.DictParameter()
-    asset_cfg = luigi.DictParameter()
+    dataset_cfg = luigi.Parameter()
```

Comment:
TODO: Should we pass dataset id and asset id instead of the actual objects? We use that pattern elsewhere so that parameters are hashable.